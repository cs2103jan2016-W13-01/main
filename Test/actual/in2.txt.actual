Task added successfully
1. incomplete; dinner with family; null; null; null
Task added successfully
1. incomplete; buy presents for dennis; null; null; null
2. incomplete; dinner with family; null; null; null
Invalid command. Please try again.
1. incomplete; buy presents for dennis; null; null; null
2. incomplete; dinner with family; null; null; null
Task not found in task list: 3
1. incomplete; buy presents for dennis; null; null; null
2. incomplete; dinner with family; null; null; null
