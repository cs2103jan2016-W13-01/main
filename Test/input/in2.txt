add dinner with family
add buy presents for dennis
delete 0
delete 3
undo
delete 2