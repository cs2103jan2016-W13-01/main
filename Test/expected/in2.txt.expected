Task added successfully: dinner with family  Once
1. incomplete; dinner with family; -; -; Once
Task added successfully: buy presents for dennis  Once
1. incomplete; buy presents for dennis; -; -; Once
2. incomplete; dinner with family; -; -; Once
Invalid command. Please try again.
Task list not loaded
Task number 3 not found in task list
1. incomplete; buy presents for dennis; -; -; Once
2. incomplete; dinner with family; -; -; Once
Action undone: add buy presents for dennis  Once
1. incomplete; dinner with family; -; -; Once
Task number 2 not found in task list
1. incomplete; dinner with family; -; -; Once